Sources/DatadogSDKTesting/Utils/PlatformUtils.swift

```diff
@@ -87,4 +88,73 @@ struct PlatformUtils {
             return (ProcessInfo.processInfo.processName, (Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String) ?? "")
         }
     }
+
+    static func getAppearance() -> String {
+        #if os(iOS) || os(tvOS) || os(watchOS)
+            let appearance: String
+            if #available(iOS 13.0, tvOS 13.0, *) {
+                switch UITraitCollection.current.userInterfaceStyle {
+                    case .unspecified:
+                        appearance = "unspecified"
+                    case .light:
+                        appearance = "light"
+                    case .dark:
+                        appearance = "dark"
+                    @unknown default:
+                        appearance = "unknown"
+                }
+            } else if #available(iOS 12.0, tvOS 12.0, *) {
+                switch UIScreen.main.traitCollection.userInterfaceStyle {
+                    case .dark:
+                        appearance = "dark"
+                    case .light:
+                        appearance = "light"
+                    case .unspecified:
+                        appearance = "unspecified"
+                    @unknown default:
+                        appearance = "unknown"
+                }
+            } else {
+                appearance = "light"
```

Comment:
Not sure if this is the best default, given that we have `"unspecified"` and `"unknown"` at our disposal.

Reply:
I chose it because we don't want to show it as a difference in that setting if a test fail because of the system version and not because of the appearance, and the backend would show it as a difference also. The appearance before iOS 12 could not be dark, so I prefered to set manually to light, does it make sense?

Reply:
I was only wondering if there's a deeper reasoning behind this default. Clearly can see: there is 👍. Sounds fine!